website_img_bg_style/views/snippets.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+     Copyright 2016 Tecnativa - Vicent Cubells
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+
+<template id="snippet_options"
+          inherit_id="website.snippet_options"
+          name="Container width type chooser">
+    <xpath expr=".">
+        <div data-snippet-option-id='background-style'
+             data-selector=".oe_img_bg">
+            <li class="dropdown-submenu">
+                <a href="#">Background style</a>
+                <ul class="dropdown-menu">
+                    <li class="dropdown-submenu">
+                        <a href="#">Position</a>
+                        <ul class="dropdown-menu">
+                            <li class="dropdown-submenu">
+                                <a href="#">Vertical</a>
+                                <ul class="dropdown-menu">
+                                    <li data-value="bg-top">
```

Comment:
Replace every `data-value` occurrence in this file by `data-select_class` to take advantage of [new features introduced in v9 for snippet options](https://www.odoo.com/documentation/9.0/howtos/themes.html#default-option-methods).

Also add an element like this at the end of every option group:

```
<li data-select_class="">
    <a>None</a>
</li>
```